Sources/AltSwiftUI/Source/Views/Controls/TextField.swift

```diff
@@ -39,12 +41,14 @@ public struct TextField<T>: View {
     ///     or not.
     ///   - onCommit: The action to perform when the user performs an action
     ///     (usually the return key) while the `TextField` has focus.
-    public init(_ title: String, value: Binding<T>, formatter: Formatter, onEditingChanged: @escaping (Bool) -> Void = { _ in }, onCommit: @escaping () -> Void = {}) {
+    public init(_ title: String, value: Binding<T>, formatter: Formatter, autocorrect: Bool = false, autocapitalize: Bool = true, onEditingChanged: @escaping (Bool) -> Void = { _ in }, onCommit: @escaping () -> Void = {}) {
```

Comment:
why don't you use the existing SwiftUI modifier methods instead of the init?

```
func disableAutocorrection(_ disable: Bool?) -> some View
func autocapitalization(_ style: UITextAutocapitalizationType) -> some View
```